R/CensoredDataSuite.R

```diff
@@ -37,12 +37,21 @@ TADA_IDCensoredData <- function(.data) {
     "TADA.ResultMeasureValueDataTypes.Flag"
   )
   TADA_CheckColumns(.data, expected_cols)
-
-  ## First step: identify censored data
-  cens <- .data %>% dplyr::filter(TADA.ResultMeasureValueDataTypes.Flag == "Result Value/Unit Copied from Detection Limit")
-  not_cens <- .data %>% dplyr::filter(!ResultIdentifier %in% cens$ResultIdentifier)
+  
+  ## Run TADA_FlagMeasureQualifierCode to add column TADA.MeasureQualifier.Flag to allow for using user-supplied Result Measure Qualifier codes to identify censored samples.
+ data_mq_flag <- TADA_FlagMeasureQualifierCode(.data)
```

Comment:
Do you think TADA_FlagMeasureQualifierCode should always run within this function, or should it only run if the TADA.MeasureQualifier.Flag column does not yet exist in the input df?

Reply:
I think it makes sense to run the function only if the TADA.MeasureQualifier.Flag column does not yet exist.

Reply:
I'll work on this